install_files/ansible-base/roles/reset-ssh-key/tasks/reset-ssh-key.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Set authorized key, removing all the authorized keys already set
+  authorized_key:
+    user: "{{ item }}"
+    key: "{{ lookup('file', '/home/amnesia/.ssh/id_rsa.pub') }}"
+    state: present
+    exclusive: True
+  with_items: "{{ ssh_users }}"
```

Comment:
The dynamic inventory check in https://github.com/freedomofpress/securedrop/blob/develop/install_files/ansible-base/inventory-dynamic#L87 will throw an error when running this task with more than one user configured in `site-specific`. Given this, is the `with_items` loop really giving us anything?

Reply:
Fair point. Updated to expect a single value.